packages/main/cypress/specs/ColorPalette.cy.tsx

```diff
@@ -44,4 +61,150 @@ describe("Color Palette tests", () => {
 				a: "1",
 			});
 	});
+
+	it("Test if selecting element works", () => {
+		cy.mount(getColorPalette());
```

Comment:
Use this as reference, with jsx we have functional components that can be used as tags themselves:

<img width="757" alt="Screenshot 2025-01-31 at 14 18 40" src="https://github.com/user-attachments/assets/2871a951-7bfa-4872-8a0b-14af144840e2" />

rework getColorPalette to `<ColorPaletteSample />`